generators/app/templates/src/utils/constants.js

```diff
@@ -15,3 +15,5 @@ export const WINDOW_WIDTH = windowDimensions.width;
 
 export const VIEWPORT_HEIGHT =
   WINDOW_HEIGHT - TABBAR_HEIGHT - ACTION_BAR_HEIGHT - (STATUS_BAR_IS_FIXED ? STATUS_BAR_HEIGHT : 0);
+
+export const IS_SMALL_DEVICE = WINDOW_HEIGHT < 570;
```

Comment:
🤔  landscape rotation might break this if supported

Reply:
@sbalay @mvbattan 
I would keep it that way until we found a better way to handle the sizes
maybe can add a comment